bin/check-snmp.rb

```diff
@@ -64,6 +64,11 @@ class CheckSNMP < Sensu::Plugin::Check::CLI
          short: '-t timeout (seconds)',
          default: '1'
 
+  option :debug,
+         short: '-D (true|false)',
+         description: 'Enable debugging to assist with inspecting OID values / data.',
+         default: 'false'
+
```

Comment:
Think we should use a boolean here, same as in https://github.com/sensu-plugins/sensu-plugins-snmp/pull/5.